README.md

```diff
@@ -700,6 +702,44 @@ In this example, all saves to `MyTable` will raise an `Dynamoid::Errors::StaleOb
 
 Calls to `update` and `update!` also increment the `lock_version`, however they do not check the existing value. This guarantees that a update operation will raise an exception in a concurrent save operation, however a save operation will never cause an update to fail. Thus, `update` is useful & safe only for doing atomic operations (e.g. increment a value, add/remove from a set, etc), but should not be used in a read-modify-write pattern.
 
+
+### Backoff strategies
+
+
+You can use several methods what do efficiently in batch mode like `.find_all` and `.import`.
+
+The backoff strategy will be used when, for any reason, some items could not be processed as part of a batch mode command.
+Operations will be re-run to process this items.
+
+Exponential backoff is the recommended way to handle throughput limits exceeding and throttling on the table.
```

Comment:
Can we also clarify that even AWS "strongly recommends that you use an exponential backoff algorithm", almost makes it feel like we should make it default rather than nil, but I can see that being too strong for some users who may just want it to retry as soon as possible.

Reply:
@richardhsu 

> but I can see that being too strong for some users

Agree

> Can we also clarify that even AWS "strongly recommends

I'm not good at writing documentation. Feel free to reword/update this section.

Reply:
We can change it to say:

> Exponential backoff is highly recommended by AWS for retrying and backoff strategies, especially when handling throughput limits being exceeded or throttled on the table.

@pboling does that seem clear to you too?